docs/navigate.rst

```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
 2. In the top right of the Visibility tab, click :guilabel:`Refresh`.
 
 The contents of the visibility table are refreshed.
+
+
+
+Configure Telemetry
+-------------------
+
+1. To view the Virtual Server statistics on the Visibility VIP Dashboard, Telemetry plugin version 1.17 or higher needs to be installed on the BIG-IP device.
```

Comment:
...VIP Dashboard, you must install the Telemetry Streaming plugin version 1.17 or higher on the BIG-IP device.